game/aamp_v2.ksy

```diff
@@ -0,0 +1,203 @@
+meta:
+  id: aamp_v2
+  title: AAMP Binary Resource Parameter Archive
+  application:
+    - Switch Toolbox
+    - aamptool
+    - aamp
+    - Wild Bits
+  file-extension:
+    - aamp
+    - bxml
+    - bas
+    - baglblm
+    - baglccr
+    - baglclwd
+    - baglcube
+    - bagldof
+    - baglenv
+    - baglenvset
+    - baglfila
+    - bagllmap
+    - bagllref
+    - baglmf
+    - baglshpp
+    - baiprog
+    - baslist
+    - bassetting
+    - batcl
+    - batcllist
+    - bawareness
+    - bawntable
+    - bbonectrl
+    - bchemical
+    - bchmres
+    - bdemo
+    - bdgnenv
+    - bdmgparam
+    - bdrop
+    - bgapkginfo
+    - bgapkglist
+    - bgenv
+    - bglght
+    - bgmsconf
+    - bgparamlist
+    - bgsdw
+    - bksky
+    - blifecondition
+    - blod
+    - bmodellist
+    - bmscdef
+    - bmscinfo
+    - bnetfp
+    - bphyscharcon
+    - bphyscontact
+    - bphysics
+    - bphyslayer
+    - bphysmaterial
+    - bphyssb
+    - bphyssubmat
+    - bptclconf
+    - brecipe
+    - brgbw
+    - brgcon
+    - brgconfig
+    - brgconfiglist
+    - bsfbt
+    - bsft
+    - bshop
+    - bumii
+    - bvege
+    - bactcapt
+  xref:
+    zeldamods: AAMP
+  endian: le
```

Comment:
All `.ksy` specs are required to specify the `/meta/license` key in order to be included into the format gallery. (If they don't, they're under [exclusive copyright](https://choosealicense.com/no-permission/) and nobody but the sole author can legally use it, copy, distribute or modify - that means you can't either, because you're not the only author I suppose.)

If you are not the sole copyright owner, you should contact all other authors and agree together on a license. We recommend [`CC0-1.0`](https://spdx.org/licenses/CC0-1.0.html) or [`MIT`](https://spdx.org/licenses/MIT.html), but any license from [this list](https://spdx.org/licenses/) can be chosen.

IMHO it's ideal to use GitHub issues (or PRs) to coordinate the consent gathering, for example see https://github.com/kaitai-io/kaitai_struct_doc/pull/30.

Reply:
I guess that closes this whole PR, because Nintendo cannot be reasoned with.

Reply:
Wait, I realized this makes me look like an idiot (which I am.) but now I realize you were probably talking about the file itself, not the format.

For licensing those, I would need both @larsenv and @HEYimHeroic to weigh in.

Reply:
> > Unfortunately, I don't think I can legally share an example file
> 
> I doubt it. Who created it?

> But I've actually found a bunch of public sample files for the `gen1_mii_wii.ksy` spec: [miicontest.wii.rc24.xyz/popular.html](https://miicontest.wii.rc24.xyz/popular.html)

The format in question was `AAMP (v2)`, which has no publicly availabe generator program that I'm aware of. (I'm working on my right now, to fix that.)

Usually these files have to be extracted from Nintendo games. The purpose for reverse-engineering them is to modify the games by re-injecting them.